pkg/fqdn/dnsproxy/proxy_test.go

```diff
@@ -1068,8 +1068,23 @@ func Benchmark_perEPAllow_setPortRulesForID(b *testing.B) {
 
 func Benchmark_perEPAllow_setPortRulesForID_large(b *testing.B) {
 	b.Skip()
+	cacheSize := 1024
+	numEPs := uint64(40)
```

Comment:
OOo, see that I doubled it to `40` here. Happy to dial it down again if you think 20 is more suitable.

Reply:
Let's keep 20 so we have a consistent baseline from which we were testing.

Reply:
Agree! Keeping the cache size and the numEPs the same as before!